tests/shell/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class FooShell(models.Model):
+    pass
+
+
+class Example(models.Model):
+    name = models.CharField(max_length=50)
```

Comment:
Let’s use names with a bit more meaning, to make the tests read better. See this post that agrees with my opinion to not use “foo”, “bar”, etc.: https://www.techyourchance.com/never-use-foo-bar-baz/

```suggestion
class Marker(models.Model):
    pass


class Person(models.Model):
    name = models.CharField(max_length=50)
```